includes/Admin/Admin_Page.php

```diff
@@ -284,6 +284,13 @@ public function render_page() {
 		$user_enabled_categories = get_user_setting( 'plugin_check_category_preferences', implode( '__', $this->get_default_check_categories_to_be_selected() ) );
 		$user_enabled_categories = explode( '__', $user_enabled_categories );
 
+		$check_repo = new Default_Check_Repository();
+
+		$non_experimental_collection = $check_repo->get_checks( Check_Repository::TYPE_ALL );
+		$complete_collection         = $check_repo->get_checks( Check_Repository::TYPE_ALL + Check_Repository::INCLUDE_EXPERIMENTAL );
+
+		$has_experimental_checks = ( count( $non_experimental_collection ) !== count( $complete_collection ) );
```

Comment:
Could this be simplified?

```php
$experimental_checks = $check_repo->get_checks( Check_Repository::INCLUDE_EXPERIMENTAL );

$has_experimental_checks = count( $experimental_checks ) > 0;

Reply:
`$experimental_checks = $check_repo->get_checks( Check_Repository::INCLUDE_EXPERIMENTAL );` <- This is not giving experimental checks.

Reply:
Ok, strange 🤔

Reply:
I think I would do this instead:

```php
$collection = $check_repo->get_checks( Check_Repository::TYPE_ALL | Check_Repository::INCLUDE_EXPERIMENTAL )->filter(
	static function ( Check $check ) {
		return $check->get_stability() === Check::STABILITY_EXPERIMENTAL;
	}
);

$has_experimental_checks = count( $collection ) > 0;
```